tests/types/test_resolvers.py

```diff
@@ -58,3 +77,20 @@ def hello2(self, info, query, limit) -> str:
         'Missing annotation for arguments "limit" and "query" '
         'in field "hello2", did you forget to add it?',
     )
+
+
+def test_raises_error_when_missing_annotation_and_resolver():
+    with pytest.raises(MissingFieldAnnotationError) as e:
+
+        @strawberry.type
+        class Query:
+            missing = strawberry.field(name="annotation")
+
+        # Avoid unused local warning
+        _ = Query
```

Comment:
It's actually PyCharm... but I didn't want to leave a PyCharm-specific pragma to ignore it. Turns out I can ignore the flake8 error code and PyCharm understands it.

I do something similar [here](https://github.com/strawberry-graphql/strawberry/blob/3a4db59859ff6e4ae5900c5e820afab7f355235b/tests/schema/test_resolvers.py#L251), but that's to get PyCharm to quit complaining that the function could be static. There's no flake8 analog afaik. In hindsight, it may be too much...

Thoughts?

Reply:
yeah, I wonder why pychamr complains about self not being used, feels a bit strange, since it is a required param on classmethod :)

Reply:
It's not exactly complaining that it's unused... moreso saying that the function should be static _because_ it's not used

Reply:
For this test, I'm _specifically_ testing against instance methods, so it felt kinda off to have PyCharm insist that the function become static